internal/playwright/config.go

```diff
@@ -141,3 +147,40 @@ func SplitSuites(p Project) (Project, Project) {
 
 	return dockerProject, sauceProject
 }
+
+func checkSupportedBrowsers(p *Project) error {
+	supportedBrwsList := []string{"chromium", "firefox", "webkit"}
+	errMsg := "browserName: %s is not supported. List of supported browsers: %s"
+
+	if p.Playwright.Params.BrowserName != "" {
```

Comment:
Browser name can be empty?

Reply:
I just tested it. This value can be empty. In this case, we need to set the browser name on the suite level.